content/en/platform/corda/4.6/enterprise/notary/notary-metrics.md

```diff
@@ -24,10 +24,11 @@ all meters use exponential moving averages.
 {{< table >}}
 
 |Metric Name|Type|Description|
-|:-----------------------------|:-----------|:-------------------------------------------------------------------|
+|:-----------------------------------|:--------------------------------|:-------------------------------------------------------------------|
 | `Commit` | Timer | Measures the time taken in milliseconds to commit a single transaction and the number of transactions per second (TPS).|
 | `IPS` | Meter | Measures the number of committed input states per second (IPS).|
-| `Rollback` | Counter | Tracks the number of database transaction rollbacks. These might occur due to transient SQL exceptions, which are mitigated by retrying, or unexpected errors that cause the notarisation to be aborted.|
+| `Rollback` (Percona only)| Counter | Tracks the number of database transaction rollbacks. These might occur due to transient SQL exceptions, which are mitigated by retrying, or unexpected errors that cause the notarisation to be aborted.|
+| `DatabaseRetry` (JPA only)| Counter | Tracks the number of attempted retries a failed transaction makes. A transaction will retry up to ten times.|
```

Comment:
```suggestion
| `DatabaseRetry` (JPA only)| Counter | Tracks the number of times the notary worker has attempted to retry committing a transaction to the database following a failed transaction. This might occur due to transient SQL exceptions, attempting to spend the same states on multiple notary workers in parallel, or due to issues with the underlying database.
```

I think this is a bit more accurate. I've also removed the reference to retrying up to 10 times, as this is only a default, and isn't relevant in the context of the metric.